Dockerfile.nginx

```diff
@@ -0,0 +1,97 @@
+# Note: This comment was in the previous dockerfile
+# Note: Not 100% sure why
+#
+# Small note, make sure to create upload/thumbs prior to running this! 
+
+#
+# builder container
+FROM docker.io/debian:bookworm-slim AS builder
+
+# install some packages
+RUN set -exu \
+  && DEBIAN_FRONTEND=noninteractive apt-get -yq update \
+  && DEBIAN_FRONTEND=noninteractive apt-get -yq install \
+    libjpeg-dev \
+    libpng-dev \
+    libfreetype6-dev \
+    software-properties-common \
+    ca-certificates \
+    lsb-release \
+    apt-transport-https \
+    curl
+
+# setup nodejs repo
+RUN set -exu \
+  && curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | apt-key add - \
+  && echo "deb https://deb.nodesource.com/node_14.x bookworm main" | tee /etc/apt/sources.list.d/nodesource.list
+
+# acquire install-php-extensions script
+RUN set -exu \
+  && curl -fsSL https://github.com/mlocati/docker-php-extension-installer/releases/download/2.1.28/install-php-extensions \
```

Comment:
Wow I never realized this was a whole thing. Why do php extensions not play nicely with docker? :thinking:

Reply:
Mainline php docker images include helper scripts that manage php extensions along with required apt/apk packages, that script is meant to be included in those to add further functionality. We don't need it as we can get extensions from repos, and it actually refuses to work on a non-php-docker-mainline base image.